src/mappings/content/utils.ts

```diff
@@ -651,3 +658,244 @@ export function encodeAssets(assets: StorageAssetsRecord | undefined): Uint8Arra
       : null
   ).toU8a()
 }
+
+export async function getFollowersAccountsForChannel(
+  overlay: EntityManagerOverlay,
+  channelId: string
+): Promise<Account[]> {
+  const followers = await overlay.getEm().getRepository(ChannelFollow).findBy({ channelId })
+
+  const followersUserIds = followers
+    .filter((follower) => follower?.userId)
+    .map((follower) => follower.userId as string)
+
+  const followersAccounts = await Promise.all(
```

Comment:
throwing promise.all might be an issue if channel has 100+ followers. Would be good to implement maximum concurrent promise number

Reply:
I have used `p-limit` to have at most 10 concurrent promises at any time.
10 is arbitrary but I think we can adjust the values in future releases if we notice any delays with notification deposits

Reply:
There are other use cases of `Promise.all` in the codebase. If you agree @attemka we can open an issue with `enhancement` label for a future release where we fix all such instances